ui/src/adapters/parsers/view.ts

```diff
@@ -425,17 +426,17 @@ export function serializeCredentialIssuance({
       data: {
         credentialSchema,
         credentialSubject: serializedSchemaForm.data === undefined ? {} : serializedSchemaForm.data,
-        expiration: credentialExpiration
-          ? serializeDate(dayjs(credentialExpiration), "date-time")
-          : null,
-        mtProof,
+        expiration: credentialExpiration ? dayjs(credentialExpiration).unix() : null,
```

Comment:
We don't need dayjs to get the timestamp from a date, just do `credentialExpiration.getTime()`

Reply:
but getTime returns timestamp with milliseconds, to request  createCredential we need to provide timestamp with seconds

Reply:
Ah! Well, we could divide by 1000, but leave it as it is.